steward/src/commands/print_config.rs

```diff
@@ -1,30 +0,0 @@
-use crate::config::StewardConfig;
-use crate::{application::APP, prelude::*};
-use abscissa_core::{Command, Clap, Runnable};
-
-#[derive(Command, Debug, Default, Clap)]
-pub struct PrintConfigCmd {
```

Comment:
I'm pretty sure we're still using and referencing `print-config` so we probably don't want to delete this file.

Reply:
`print-config` is using `config-cmd.rs`, maybe a mistake but that's how it's been working.